examples_ch5xx/lowpower/funconfig.h

```diff
@@ -3,12 +3,12 @@
 
 #define FUNCONF_USE_HSI           0 // CH5xx does not have HSI
 #define FUNCONF_USE_HSE           1
-#define CLK_SOURCE_CH5XX          CLK_SOURCE_PLL_60MHz // default so not really needed
-#define FUNCONF_SYSTEM_CORE_CLOCK 60 * 1000 * 1000     // keep in line with CLK_SOURCE_CH5XX
+#define CLK_SOURCE_CH5XX          CLK_SOURCE_PLL_100MHz // default so not really needed
+#define FUNCONF_SYSTEM_CORE_CLOCK 100 * 1000 * 1000     // keep in line with CLK_SOURCE_CH5XX
 
 #define FUNCONF_DEBUG_HARDFAULT   0
 #define FUNCONF_USE_CLK_SEC       0
 #define FUNCONF_USE_DEBUGPRINTF   0 // saves 16 bytes, enable / remove if you want printf over swio
 #define FUNCONF_INIT_ANALOG       0 // ADC is not implemented yet
-
+#define POWERED_BY_V5PIN          0 // Set this to 1 if you use CH570/2 and use 5V to power it.
```

Comment:
maybe it should start with FUNCONF_ like the rest.

Reply:
Would it not make more sense to just have this a bit flag that should be ored with the power plan in the examples? Some applications might dynamically switch power supplies at runtime.